tspserver/src/main/resources/application.properties

```diff
@@ -0,0 +1,11 @@
+server.port=8086
+server.contextPath=/myapp
+
+spring.main.banner-mode=off
+logging.level.org.springframework=ERROR
+
+spring.datasource.platform=postgres
```

Comment:
Weź dodaj do readme może jakiej wersji packi do postgresa potrzebujemy.

Reply:
Dodałem do readme wersję postgresa, jeśli to miałeś na myśli mówiąc "paka". Bo jeśli chodziło ci o wersję sterownika do tej bazy danych, to jest ona najnowsza i jest pobierana przez mavena (plik pom.xml) więc żadnych sterowników nie potrzebujecie, maven sam wam je zassie